packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGenerator.ts

```diff
@@ -81,11 +81,15 @@ export class ThemeGenerator {
   /*
    * Gets code-formatted load theme blob that can be copy and pasted.
    */
-  public static getThemeAsCode(slotRules: IThemeRules): any {
+  public static getThemeAsCode(slotRules: IThemeRules, forNewThemeDesigner: boolean): any {
```

Comment:
Would be nice not to add boolean flags, since it suggests the caller knows something about the implementation of this method already. Can this be a separate method, like `getThemeAsCodeForThemeDesigner` ?

Reply:
Done.